src/main/webapp/components/Header.js

```diff
@@ -1,12 +1,20 @@
 import * as React from "react";
+import { Navigation } from "./Navigation";
 
 /**
  * Main Navigation Component
  */
 export const Header = () => {
+  const TITLE = 'LLVM Build View';
+
   return (
-    <header>
-    LLVM Build View
+    <header className='header'>
+      <div>
+        <span className='title'>{TITLE}</span>
```

Comment:
Consider to just inline the title text directly into the span, if possible, for now. (if we need to share this text between separate bits in the future, that would be a good reason to extract it as a title.)

Reply:
Understood. I originally wrapped in a span for styling purposes. If this was inline, I'm not sure It would have the same flexibility.